tools/bigquery-antipattern-tool/src/main/java/com/pso/bigquery/optimization/Main.java

```diff
@@ -8,30 +8,38 @@
 
 import static com.pso.bigquery.optimization.util.ZetaSQLHelperConstants.*;
 import org.apache.commons.cli.*;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
 
 public class Main {
+    public static String getInputQuery(String[] args)throws ParseException, IOException{
 
-    public static void main(String[] args) throws ParseException {
-
-        String query_str =
-                "\n" +
-                        "SELECT " +
-                        "   t1.*, " +
-                        "   t2.col2 " +
-                        "FROM \n" +
-                        "  (SELECT * FROM `project.dataset.table1`) t1\n" +
-                        "LEFT JOIN\n" +
-                        "  `project.dataset.table2` t2\n ON t1.col1 = t2.col2";
+        String query_str = "\n" +
```

Comment:
nit. The common practice in Java is to use camelCase instead of snake_case. 
```suggestion
        String queryStr = "\n" +
```